features/event-publisher/org.wso2.carbon.event.output.adapter.server.feature/src/main/resources/conf_templates/org.wso2.carbon.event.output.adapter.default.json

```diff
@@ -9,5 +9,7 @@
   "output_adapter.email.hostname": "smtp.gmail.com",
   "output_adapter.email.port": "587",
   "output_adapter.email.enable_start_tls": true,
-  "output_adapter.email.enable_authentication": true
-}
\ No newline at end of file
+  "output_adapter.email.enable_authentication": true,
+  "output_adapter.email.signature": "$ref{output_adapter.email.username}",
+  "output_adapter.email.replyTo": "$ref{output_adapter.email.from_address}"
```

Comment:
Change the name of the key {{output_adapter.email.reply_to}}

Reply:
done